### ISSUE TITLE
Feature Request: Add Support for File Services

### ISSUE THREAD

**Issue by prof-m** [hydrusvideodeduplicator/hydrus-video-deduplicator#24]
tl;dr - profm writes a feature request for a PR ( #25 ) they're about to put up, because, uh, documentation or something

# Summary
Add functionality for hydrus-video-deduplicator to target specific file domains within a given client

# Terminology
- I mildly hate that I have to add this section, but afaik, 'file domain' == 'file service'

# Background

- Hydrus supports the concept of [multiple local file domains](https://hydrusnetwork.github.io/hydrus/advanced_multiple_local_file_services.html)
- hydrus-video-deduplicator currently only gets files from the virtual 'all local files' domain
- (in reality, probably only a few of us use the multiple local file domain functionality of hydrus, but i'm one of them and wanted to take a crack at adding something simple in here)

# Request
Add a new option to the CLI that allows a user to supply a specific file service key from Hydrus, and have hydrus-video-deduplicator only query for files from that file service when said CLI option is supplied.

# Considerations
### Duplicate find results caching

- Ideally, specifying a file service does not prevent the caching of duplicate file results, and can utilize any pre-existing cache of duplicate file results. 
- Since the default file service is 'all local files', any execution that uses a specific file service is going to be a subset of the default file service anyways

### Specifying Multiple File Services in a Single Execution
 - Let's not do this, because 1) MVP, and 2) [HyDev says unioning local file services can make for reeeeallly slow queries](https://hydrusnetwork.github.io/hydrus/developer_api.html#parameters_file_domain)

**Comment by appleappleapplenanner** [comment]
Added in 48241d7
